src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -766,22 +778,73 @@ export class StudyViewPageStore {
 
     // all queried studies, includes both physcial and virtual studies
     // this is used in page header(name and description)
-    @computed get queriedStudies() {
-        let filteredVirtualStudies = _.map(this.filteredVirtualStudies.result, virtualStudy => {
-            return {
-                name: virtualStudy.data.name,
-                description: virtualStudy.data.description,
-                studyId: virtualStudy.id,
-            } as CancerStudy;
-        });
+    readonly displayedStudies = remoteData({
+        await: () => [this.filteredVirtualStudies, this.filteredPhysicalStudies, this.queriedPhysicalStudies],
+        invoke: async () => {
+            if (this.filteredPhysicalStudies.result.length === 0 && this.filteredVirtualStudies.result.length === 1) {
+
+                const virtualStudy = this.filteredVirtualStudies.result[0]
+                return [{
+                    name: virtualStudy.data.name,
+                    description: virtualStudy.data.description,
+                    studyId: virtualStudy.id,
+                } as CancerStudy]
+            }
+            return this.queriedPhysicalStudies.result;
+        },
+        default: []
+    });
 
-        return [...this.filteredPhysicalStudies.result, ...filteredVirtualStudies];
-    }
+    //get originStudies when there is only one study in query and that too virtual study
```

Comment:
we need a better description of what originStudies is in terms of business logic.  the name doesn't really tell me.